src/containers/image-upload-container/image-upload-container.js

```diff
@@ -16,7 +16,7 @@ const ImageUploadContainer = ({
       <label htmlFor='upload-photo'>
         <input
           style={{ display: 'none' }}
-          accept='image/*'
+          accept='video/*,image/*'
           id='upload-photo'
```

Comment:
Why this element have id `upload-photo` if you are using photo and video upload?